packages/react-core/src/components/NotificationBadge/examples/NotificationBadgeWithCount.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NotificationBadge, NotificationBadgeVariant } from '@patternfly/react-core';
+
+export const NotificationBadgeWithCount: React.FunctionComponent = () => {
+  const [firstVariant, setFirstVariant] = React.useState(NotificationBadgeVariant.unread);
+  const [firstVariantCount, setFirstVariantCount] = React.useState(30);
+  const [secondVariant, setSecondVariant] = React.useState(NotificationBadgeVariant.attention);
+  const [secondVariantCount, setSecondVariantCount] = React.useState(30);
+
+  const onFirstClick = (value: NotificationBadgeVariant) => {
+    setFirstVariant(value);
+    setFirstVariantCount(90);
```

Comment:
Why 90? Just seems weird to see it go from 30 to 90 when marking read. I'd think if we are going to change it, perhaps 0 would make more sense.

Reply:
The existing example set the exact same value (30), so I was just doing a different number.  More notifications could have come in??